### ISSUE TITLE
Nan elbo loss

### ISSUE THREAD

**Issue by youonlytrackonce** [ostadabbas/DeepPBM#3]
Hello,

thank you for your great job.

I have been training video 1 but after some epoch elbo loss becomes “nan”.  My configuration is default (epoch number, batch size, learning rate. My environment is in miniconda, python 3.6, pytorch 1.0. Os is ubuntu 20.04, gpu is gtx 1080ti

I have one more pc that has the same hardware. I replicate the same experiment and this time elbo loss stays constant and after the training the resulting images are garbage.

what would you suggest me to do to have right training configuration?

Thanks
Fatih

**Comment by helenypzhang** [comment]
Hey, Fatih,

I am trying to implement this paper but I could not find the BMC2012 dataset.
Do you mind sharing this dataset?
My email is ypzhang5-c@my.cityu.edu.hk

Many thanks and Best Regards,
ZHANG Yupei

**Comment by gone2808** [comment]
try to reduce learning rate.
